Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormCompanyActQuestion.cshtml.cs

```diff
@@ -13,7 +14,7 @@ public class LegalFormCompanyActQuestionModel(
     ITempDataService tempDataService) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```

Comment:
Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors

Reply:
Done